python/lsst/ts/hexrotcomm/command_telemetry_server.py

```diff
@@ -329,6 +364,35 @@ async def write_config(self):
         header, curr_tai = self.update_and_get_header(self.config.FRAME_ID)
         await tcpip.write_from(self.telemetry_server.writer, header, self.config)
 
+    async def write_command_status(self, counter, status, duration=0, reason=""):
+        """Write a command status.
+
+        Note that this is written to the command socket.
+
+        Parameters
+        ----------
+        counter : `int`
+            counter field of command being acknowledged.
+        status : `CommandStatusCode`
+            Command status code.
+        duration : `float` or `None`
+            Estimated duration. None is treated as 0.
+        reason : `str`
+            Reason for failure. Should be non-blank if and only if the
+            command failed.
```

Comment:
These two arguments are "optional". And please consider to put the default values in doc string.

Reply:
I added ", optional" to the docs. Sphinx shows the default values in the call signature and I hate duplicating information (because it can so easily go out of date as the code evolves) so I only document the defaults when there is something non-obvious about them.